Builds/ArchLinux/PKGBUILD

```diff
@@ -25,9 +25,6 @@ build() {
 }
 
 check() {
-	cd "$srcdir/$pkgname"
```

Comment:
I'm not familiar with Arch Linux / PKGBUILD, but I think you are right to keep the `cd`